Documentation/welcome.md

```diff
@@ -1,7 +1,14 @@
 Welcome to the MonoGame game library documentation hub.
 
-### WORK IN PROGRESS
-Note that the MonoGame documentation project is currently a work in progress and applies to the **latest development build** of MonoGame.
+This area of the site contains the documentation on the API of MonoGame as well as how to use it to create great games.
+Note that this is a work in progress so there will be gaps in the documentation coverage. 
+
+If you cannot find what you need here you can also look at the [Microsoft XNA documentation](). MonoGame is API compatible
```

Comment:
Link to XNA docs: https://msdn.microsoft.com/en-us/library/bb200104.aspx
And API reference: https://msdn.microsoft.com/en-us/library/bb203940.aspx
Not sure which is more appropriate.